src/main/java/de/numcodex/sq2cql/model/structured_query/RangeCriterion.java

```diff
@@ -1,33 +1,38 @@
 package de.numcodex.sq2cql.model.structured_query;
 
+import static java.util.Objects.requireNonNull;
+
 import de.numcodex.sq2cql.Container;
 import de.numcodex.sq2cql.model.Mapping;
 import de.numcodex.sq2cql.model.MappingContext;
+import de.numcodex.sq2cql.model.common.TermCode;
 import de.numcodex.sq2cql.model.cql.BetweenExpression;
 import de.numcodex.sq2cql.model.cql.BooleanExpression;
 import de.numcodex.sq2cql.model.cql.Expression;
+import de.numcodex.sq2cql.model.cql.FunctionInvocation;
 import de.numcodex.sq2cql.model.cql.IdentifierExpression;
 import de.numcodex.sq2cql.model.cql.InvocationExpression;
 import de.numcodex.sq2cql.model.cql.QuantityExpression;
 import de.numcodex.sq2cql.model.cql.TypeExpression;
-
 import java.math.BigDecimal;
 import java.util.List;
 import java.util.Optional;
 
-import static java.util.Objects.requireNonNull;
-
 /**
- * A {@code RangeCriterion} will select all patients that have at least one resource represented by that concept and
- * a range of numeric values.
+ * A {@code RangeCriterion} will select all patients that have at least one resource represented by
+ * that concept and a range of numeric values.
  * <p>
- * Examples are {@code Observation} resources representing the concept of a numeric laboratory value.
+ * Examples are {@code Observation} resources representing the concept of a numeric laboratory
+ * value.
  */
 public final class RangeCriterion extends AbstractCriterion {
 
-    private final BigDecimal lowerBound;
-    private final BigDecimal upperBound;
-    private final String unit;
+  static final TermCode AGE = TermCode.of("http://snomed.info/sct", "424144002",
```

Comment:
Please move that in the class with the function name switch and use the English version: "Current chronological age"